### DISCUSSION TITLE
SOC User Permissions

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#6696] [General] [answered]
Hi,
I have 3 SOC users with different logic roles, I know that version 2.3.80 has the option to add a role to every user based on the permissions we want it to have.

The problem is that I have version 2.3.50 installed. Is there a way to give permissions to the SOC users in a different way that works on that version?

Another solution that would work if possible is to update the version, Is there a quick way to update the version offline without downloading the whole Image all over again?

Thank you.

**Comment by dougburks** [comment, marked as answer]
We highly recommend that you upgrade to the latest Security Onion 2.3.91 as it includes the features you're looking plus additional security fixes and bug fixes:
https://docs.securityonion.net/en/2.3/soup.html

**Reply by yula-21** [reply]
Thank you for your reply.

I just have one more question. When using soup on an Airgap, it updates the version and keeps all of my configuration?
Or will it reset everything Ive done?

**Reply by dougburks** [reply]
Yes, soup updates the version and keeps your configuration.
